src/integrated/land.jl

```diff
@@ -526,6 +523,36 @@ function soil_boundary_fluxes!(
         ) +
         p.excess_heat_flux +
         p.snow.snow_cover_fraction * p.ground_heat_flux
+
+    (; ρc_ds, earth_param_set) = soil.parameters
+    _σ = LP.Stefan(earth_param_set)
+
+    T_sfc = ClimaLand.Domains.top_center_to_surface(p.soil.T)
+    ρc = p.soil.sub_sfc_scratch
+    @. ρc =
+        volumetric_heat_capacity(p.soil.θ_l, Y.soil.θ_i, ρc_ds, earth_param_set)
+    ρc_sfc = ClimaLand.Domains.top_center_to_surface(ρc)
```

Comment:
I think this is allocating - is there another scratch space we could use?

Reply:
oh! I think top_center_to_surface is OK - because it is just pulling out a view of the field.

Reply:
hmm I see